src/ua.c

```diff
@@ -1965,3 +2035,18 @@ int uag_set_extra_params(const char *eprm)
 
 	return 0;
 }
+
+
+void ua_set_custom_hdrs(struct ua *ua, struct list custom_headers)
```

Comment:
the custom_headers parameter should be of type pointer to a const struct list:

`const struct list *custom_headers`

it is not recommended to pass structs like this as function parameters.